### PR TITLE
Ported theme from polykey.io

### PR DESCRIPTION
**Pull request by bmamouri**
Related Issue: #6 

I've added Matrix AI's default linter and the prettier setup that we have used for Matrix AI's website. That's the reason for the huge number of file changes. Otherwise, this PR is rather small.

The PR includes the fonts and the new header. It also changes the footer to matches the look & feel of polykey.io.

### CODE REVIEW

**Comment by bmamouri** [comment]
@amydevs, can you please do me the favour of looking into this PR as well? Thank you

**Comment by CMCDragonkai** [comment]
@amydevs @bmamouri how does this PR work with the previous PR #12?

Does there need to be some cherry-picking between the 2?

I'm a bit confused about how to review this, and if we are missing any work in #12.

If we **aren't** please close #12 and indicate that it is superseded by this PR.

**Comment by bmamouri** [comment]
@CMCDragonkai, I am going to close #12, because it is an incomplete piece of work and doesn't work properly.

I've also moved anything useful from #12 to this PR. So you are good to only review this PR. Thanks

**Comment by bmamouri** [comment]
@CMCDragonkai, by the way, the reason for the sheer amount of changes in this PR is that I've added the prettier (similar to other Polykey repos), and reformatted all the files.

**Comment by CMCDragonkai** [comment]
Cool thanks.

**Comment by CMCDragonkai** [comment]
Note that unlike the websites, the wiki is going to have alot of files and photos.

These should go into github LFS. See comment here: https://github.com/MatrixAI/Polykey-Docs/pull/12#issuecomment-1373063144#17

This will still work when deployed to cloudflare, the CI/CD should then download everything when pushing up. Make sure to configure LFS in the gitlab CI file.

**Comment by CMCDragonkai** [comment]
@amydevs any changes necessary from polykey.io review?

Also don't forget the matrix.ai hubspot change over too.

**Review by amydevs — APPROVED** [review]
Everything looks good, should be ready to merge.

**Comment by CMCDragonkai** [comment]
@amydevs did you test the worker system? And also remember our developement worker URL, can you deploy and send me the link on slack for inspection?

File: package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "start": "docusaurus start",
     "swizzle": "docusaurus swizzle",
     "build": "docusaurus build --out-dir=./public",
-    "deploy": "wrangler publish"
+    "deploy": "wrangler publish",
+    "code:format": "prettier --write \"**/*.{ts,tsx,js,jsx,css,md,mdx}\""
```
Comment (CMCDragonkai) [inline package.json:21]: This should be `lintfix` like the rest.

File: server/index.ts [hunk 2]
```diff
@@ -1,67 +1,49 @@
...
- */
-function mapRequestToDocs(req: Request): Request {
-  // Default mapping resolves directory URLs e.g. `/dir` becomes `/dir/index.html`
-  const assetRequest = mapRequestToAsset(req);
-  const assetUrl = new URL(assetRequest.url);
-  // Strip the `/docs` segment: `https://polykey.io/docs/...` -> `https://polykey.io/...`
-  assetUrl.pathname = assetUrl.pathname.replace(/^\/docs/, '/');
-  return new Request(assetUrl.toString(), assetRequest);
-}
-
```
Comment (CMCDragonkai) [inline server/index.ts:64]: This removal breaks production.

**Pull request merged**